src/backend/src/app.js

```diff
@@ -1,61 +1,83 @@
 import debug from 'debug'
 import { createRequire } from 'module'
-const rnBridge = createRequire(import.meta.url)('rn-bridge')
+const require = createRequire(import.meta.url)
+/** @type {import('../types/rn-bridge.js')} */
+const rnBridge = require('rn-bridge')
 import { MapeoManager } from '@mapeo/core'
-import { createMapeoServer } from '@mapeo/ipc'
 import { KeyManager } from '@mapeo/crypto'
 import RAM from 'random-access-memory'
 
-import MessagePortLike from './message-port-like.js'
+import { Server } from './server.js'
 
-/** @typedef {Object} State
- *
- * @property {import('../types/api.js').Status} status
- */
-
-// TODO: Account for args passed from node.startWithArgs
-debug.enable('*')
-const log = debug('mapeo:index')
-
-log('Starting Node...')
+const log = debug('mapeo:app')
 
 /**
  * @param {Object} [options]
  * @param {string} [options.version] Device Version
  */
 export async function init({ version } = {}) {
-  log(`Device version is ${version}`)
+  // TODO: Account for args passed from node.startWithArgs
+  debug.enable('*')
 
-  // 1. Initialize server state
-  /** @type {State} */
-  const state = { status: 'idle' }
+  log('Starting app...')
+  log(`Device version is ${version}`)
 
-  // 2. Initialize Mapeo API server
-  const channel = new MessagePortLike()
+  // TODO: Persisted and read from local file
+  /** @type {{ activeProjectId: string | null }} */
+  const state = {
+    activeProjectId: null,
+  }
 
+  // 1. Initialize Mapeo
   const manager = new MapeoManager({
     rootKey: KeyManager.generateRootKey(),
+    // TODO: Use actual file storage instead of memory
     dbFolder: ':memory:',
     coreStorage: () => new RAM(),
   })
 
-  const { close } = createMapeoServer(manager, channel)
+  // Automatically create initial project if no projects exist yet
+  if ((await manager.listProjects()).length === 0) {
+    const projectId = await manager.createProject()
+    log(`Created initial project with id ${projectId}`)
+    state.activeProjectId = projectId
```

Comment:
i know that I said storing the projectId in memory was fine for now...but Im realizing that it would be good to test having things persist. So can you persist the project id before we merge.